src/Subscriptions/Storage/RedisStorageManager.php

```diff
@@ -97,8 +97,8 @@ public function storeSubscriber(Subscriber $subscriber, string $topic): void
         $this->connection->command('set', array_merge([
             $this->channelKey($subscriber->channel),
             $this->serialize($subscriber),
-            // ...and set the EX option to set the ttl in one command.
-        ], $this->ttl ? ['EX', $this->ttl] : []));
+            // ...and set the TTL in one command.
+        ], $this->ttl ? [$this->ttl] : []));
```

Comment:
My guess is that this should have been `['EX' => $this->ttl]`. Do we have a test case to cover this?

Reply:
No, explanation in the comment. 'ex' => ttl is a common syntax (so good guess^^), but would break the code aswell, because in \Illuminate\Redis\Connections\Connection::command the framework uses array spread to pass parameters and that does not work for associative arrays.

Reply:
You could do `[ 'key', 'value', ['ex' => $ttl] ]` (double wrapping the 'ex' part) techincally to set more options for the timeout, because Redis::set also can take arguments like this:

```php
// Will set the key, if it doesn't exist, with a ttl of 10 seconds
$redis->set('key', 'value', ['nx', 'ex' => 10]);

// Will set a key, if it does exist, with a ttl of 1000 milliseconds
$redis->set('key', 'value', ['xx', 'px' => 1000]);
```

Reply:
> Do we have a test case to cover this?

I guess I did not add a ttl when integration testing.